internal/staticanalysis/parsing/js/js_parsing.go

```diff
@@ -103,8 +104,8 @@ func ParseJS(parserConfig ParserConfig, filePath string, sourceString string) (r
 
 	// convert the elements into more natural data structure
 	for _, element := range storage {
-		switch element.SymbolType {
-		case "Identifier":
+		switch parsing.SymbolType(element.SymbolType) {
```

Comment:
Can we just set the `SymbolType` type in the struct to be `parsing.SymbolType` ?

Reply:
I can try - `parsing.SymbolType is populated by `json.Unmarshal` so I wasn't sure if it would work with custom types